commands/releases/status_helper.js

```diff
@@ -1,25 +1,35 @@
 'use strict'
 
-module.exports = function (s) {
+function pendingDescription (release, runningRelease, runningSlug) {
+  if (
```

Comment:
seems like this could just be:

```js
 if (runningRelease && runningRelease.id && runningSlug.process_types && runningSlug.process_types.release)
```